content/intro-to-storybook/react-native/pt/simple-component.md

```diff
@@ -0,0 +1,187 @@
+    "@storybook/addon-ondevice-actions",
+  ],
+};
+```
+
+Se você verificar a propriedade `stories`, verá que o Storybook está procurando histórias na pasta de `components`.
+
+No React Native Storybook, usamos a configuração em `main.js` para gerar um arquivo chamado `storybook.requires.js`, porque o React Native ainda não suporta importações dinâmicas. Este arquivo é gerado quando você executa `yarn storybook` para iniciar o storybook ou `yarn storybook-generate` se desejar apenas gerar novamente o arquivo `storybook.requires.js`. Este arquivo é usado para carregar todas as histórias do projeto e também importar seus addons, sempre que você achar que uma história não está sendo carregada, você pode tentar regenerar este arquivo.
+
+Agora vamos criar o componente de tarefa e o arquivo de história que o acompanha: `components/Task.jsx` e `components/Task.stories.jsx`.
+
+Começaremos com uma implementação básica da `Task`, simplesmente pegando os atributos que sabemos que precisaremos e as duas ações que você pode realizar em uma tarefa (para movê-la entre as listas):
+
+```jsx:title=components/Task.jsx
+import { StyleSheet, TextInput, View } from "react-native";
+import { styles } from "./styles";
+
+export const Task = ({
+  task: { id, title, state },
+  onArchiveTask,
+  onPinTask,
+}) => {
+  return (
+    <View style={styles.listItem}>
+      <TextInput value={title} editable={false} />
+    </View>
+  );
+};
+```
+
+Agora adicione o arquivo da história:
+
+```jsx:title=components/Task.stories.jsx
+import { Task } from "./Task";
+
+export default {
+  title: "Task",
+  component: Task,
+  argTypes: {
+    onPinTask: { action: "onPinTask" },
+    onArchiveTask: { action: "onArchiveTask" },
+  },
+};
+
+export const Default = {
+  args: {
+    task: {
+      id: "1",
+      title: "Test Task",
+      state: "TASK_INBOX"
+    },
+  },
+};
+
+export const Pinned = {
+  args: { task: { ...Default.args.task, state: "TASK_PINNED" } },
+};
+
+export const Archived = {
+  args: { task: { ...Default.args.task, state: "TASK_ARCHIVED" } },
+};
+```
+
+Em nossos arquivos de histórias, usamos uma sintaxe chamada Component Story Format (CSF). Neste caso, estamos usando o CSF3, que é uma nova versão atualizada do CSF ​​compatível com as versões mais recentes do Storybook. Nesta versão do CSF, há muito menos clichê, o que facilita o início.
+
+There are two basic levels of organization in Storybook: the component and its child stories. Think of each story as a permutation of a component. You can have as many stories per component as you need.
+
+Existem dois níveis básicos de organização no Storybook: o componente e suas histórias filhas. Pense em cada história como uma permutação de um componente. Você pode ter quantas histórias por componente precisar.
+
+- **Componente**
+  - História
+  - História
+  - História
+
+Para informar ao Storybook sobre o componente que estamos documentando, criamos uma exportação padrão `default` que contém:
+
```

Comment:
@Wanderson77 can you adjust this to be: "De forma a informar o Storybook acerca do componente que está a ser documentado, é criado um default export que contém:"